alerter.js

```diff
@@ -1,26 +1,35 @@
+const { expect } = require('chai');
+
 let alertFailureCount = 0;
+let thresholdTemperature = 200;
 
 function networkAlertStub(celcius) {
-    console.log(`Alert! Temperature is ${celcius} degrees`);
-    // Return 200 for ok
-    // Return 500 for not-ok
-    // stub always succeeds and returns 200
-    return 200;
+	console.log(`Alert! Temperature is ${celcius} degrees`);
+	// Return 200 for ok
+	// Return 500 for not-ok
+	// stub always succeeds and returns 200
+	if (celcius < thresholdTemperature) {
+		return 200;
+	} else {
+		return 500;
+	}
 }
 
 function alertInCelcius(farenheit) {
-    const celcius = (farenheit - 32) * 5 / 9;
-    const returnCode = networkAlertStub(celcius);
-    if (returnCode != 200) {
-        // non-ok response is not an error! Issues happen in life!
-        // let us keep a count of failures to report
-        // However, this code doesn't count failures!
-        // Add a test below to catch this bug. Alter the stub above, if needed.
-        alertFailureCount += 0;
-    }
+	const celcius = ((farenheit - 32) * 5) / 9;
+	const returnCode = networkAlertStub(celcius);
+	if (returnCode != 200) {
+		// non-ok response is not an error! Issues happen in life!
+		// let us keep a count of failures to report
+		// However, this code doesn't count failures!
+		// Add a test below to catch this bug. Alter the stub above, if needed.
+		alertFailureCount += 0;
+	}
 }
 
 alertInCelcius(400.5);
 alertInCelcius(303.6);
 console.log(`${alertFailureCount} alerts failed.`);
 console.log('All is well (maybe!)');
+expect(alertFailureCount).equal(1);
+expect(alertInCelcius(400.5)).to.throw(Error);

```

Comment:
This test will fail even after you fix the code. See https://stackoverflow.com/questions/21587122/mocha-chai-expect-to-throw-not-catching-thrown-errors